installing/getting-started-rhacs-cloud-ocp.adoc

```diff
@@ -119,4 +119,30 @@ If no data appears in the ACS Console:
 * Ensure that at least one secured cluster is connected to your {product-title-managed-short} instance. For more information, see the "Installing secured cluster resources on each cluster" section.
 * Examine your Sensor pod logs to ensure that the connection to your {product-title-managed-short} instance is successful.
 * In the {ocp-short} cluster, navigate to *Platform Configuration* -> *Clusters* to verify that the components are healthy and view additional operational information.
-* Examine the values in the `SecuredCluster` API in the Operator on your local cluster to ensure that the *Central API Endpoint* has been entered correctly. This value should be the same value as shown in the *ACS instance* details in the {cloud-console}.
\ No newline at end of file
+* Examine the values in the `SecuredCluster` API in the Operator on your local cluster to ensure that the *Central API Endpoint* has been entered correctly. This value should be the same value as shown in the *ACS instance* details in the {cloud-console}.
+
+[id="default-access-acs-console"]
+== Default access to ACS Console
+
+By default, the only authentication mechanism available to users is authentication via Red Hat SSO.
+Red Hat SSO auth provider cannot be deleted or changed.
+However, you can change minimum access role and add additional rules.
+
+[NOTE]
+====
+In case you want more control over access to your ACS console, it is recommended not to rely on default auth provider.
+You can set up your own IdP provider right after provisioning using one of the guides under `Operating -> Managing user access`
+====
+
+Default auth provider has required attribute `rh_org_id` set to the organisation ID assigned to your account.
+Only users from your organisation can access ACS console via Red Hat SSO auth provider.
+
+Minimum access role is set to `None`. Assigning different value to this field will give access to ACS instance
+to all users within organisation.
+
+Other rules set up in default auth provider:
+
+* Rule mapping your `userid` to `Admin`
+* Rules mapping administrators of organisation to `Admin`
+
+You can add more rules to grant access to ACS console to someone else in your organization, for example, using `email` as a key.
```

Comment:
```suggestion
You can add more rules to grant access to the {product-title-short} web console to someone else in your organization; for example, by using `email` as a key.
```

Reply:
@rukletsov I wouldn't advertise using `groups` in `sso.redhat.com` case - mainly because I don't know any beneficial use case except for `admin:org:all`. I advertised `email` as a convenient way to grant access to a single user - to give access to a group of users I would use some other identity provider I have more control over.

Reply:
Aha, good point. I'd then add a note saying that in case of own IdP we advice to use `groups` for portability.